src/sagas/import-project.saga.js

```diff
@@ -97,14 +97,19 @@ export function* importProject({ path }: Action): Saga<void> {
     // icon.
     // TODO: Try importing the existing project's favicon as icon instead?
     const color = yield call(getColorForProject, json.name);
+
+    // If guppy key already exists in package.json (if importing a project that was
+    // on Guppy previously), then we don't want to overwrite guppy.name because
+    // it's the proper spaced version of the project name (json.name is dashed)
+    const hasGuppyKey = json.hasOwnProperty('guppy');
```

Comment:
Hm, I wonder if it makes sense to just use the `guppy` field entirely, if it already exists...

```js
// If guppy key already exists in package.json (if importing a project that was
// on Guppy previously), then we don't want to overwrite guppy.name because
// it's the proper spaced version of the project name (json.name is dashed)
const hasGuppyKey = json.hasOwnProperty('guppy')

const guppyFields = hasGuppyKey
  ? json.guppy
  : { /* All the normal stuff */ };

const packageJsonWithGuppy = {
  ...json,
  guppy: guppyFields,
};
```

The only field I'm not sure if we should re-create is `createdAt`, but if it was already in Guppy previously, I think it probably makes sense to preserve it; the project isn't being re-created.

Reply:
I think this probably makes sense rather than only persisting some guppy info and not others. Plus the more I think about @superhawk610's color comment, I realize it might be seen if the original project used color in the first place (originally imported) and got used to that color. So I think let's persist everything.